SmartPlaces.Facilities/samples/Deploy/sampleProjects/topology.bicep

```diff
@@ -742,6 +742,71 @@ resource dashboard 'Microsoft.Portal/dashboards@2020-09-01-preview' = {
               }
             }
           }
+          {
+            position: {
+              x: 18
+              y: 10
+              rowSpan: 4
+              colSpan: 6
+            }
+            metadata: {
+              inputs: [
+                {
+                  name: 'options'
+                  isOptional: true
+                }
+                {
+                  name: 'sharedTimeRange'
+                  isOptional: true
+                }
+              ]
+              #disable-next-line BCP036
+              type: 'Extension/HubsExtension/PartType/MonitorChartPart'
+              settings: {
+                content: {
+                  options: {
+                    chart: {
+                      metrics: [
+                        {
+                          resourceMetadata: {
+                            id: appInsights.id
+                          }
+                          name: 'UpdateDigitalTwinAsyncSkipped'
+                          aggregationType: 1
+                          namespace: 'azure.applicationinsights'
+                          metricVisualization: {
+                            displayName: 'UpdateDigitalTwinAsyncSkipped'
+                            resourceDisplayName: appInsightsName
+                          }
+                        }
+                      ]
+                      title: 'Update Twin Skipped'
+                      titleKind: 2
+                      visualization: {
+                        chartType: 1
```

Comment:
Should this be chartType 5 with grouping since the dimension was added in the ADTGraphManager?